transport/internet/ws/hub.go

```diff
@@ -69,12 +70,25 @@ func (wsl *WSListener) listenws(address v2net.Address, port v2net.Port) error {
 	errchan := make(chan error)
 
 	listenerfunc := func() error {
-		return http.ListenAndServe(address.String()+":"+strconv.Itoa(int(port.Value())), nil)
+		ol, err := net.Listen("tcp", address.String()+":"+strconv.Itoa(int(port.Value())))
```

Comment:
Use ListenTCP here. It removes some unnecessary error checking in NewStoppableListener.